backend/routes/upload_reservation_routes.py

```
@@ -0,0 +1,120 @@
+                is_valid, text = validate_document(filepath)
+                validation_results[key] = {"valid": is_valid, "extracted_text": text}
+                if not is_valid:
+                    raise BadRequest(f"Invalid document for {key}")
+            else:
+                raise BadRequest(f"Invalid file type for {key}")
+
+        # Process selfie
+        selfie = request.files['selfie']
+        if selfie and allowed_file(selfie.filename):
+            selfie_filename = f"{form_data['name']}-{form_data['surname']}-{cf}-selfie.jpg"
+            selfie_filepath = save_file(selfie, reservation_folder, selfie_filename)
```

Ruff (0.8.2)

Local variable `selfie_filepath` is assigned to but never used

Remove assignment to unused variable `selfie_filepath`

(F841)